pkgs/by-name/cb/cbmc/package.nix

```diff
@@ -10,19 +10,36 @@
   flex,
   makeWrapper,
   perl,
+  substituteAll,
+  substitute,
+  cudd,
+  fetchurl,
+  nix-update-script,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "cbmc";
-  version = "6.0.1";
+  version = "6.4.0";
 
   src = fetchFromGitHub {
     owner = "diffblue";
-    repo = pname;
-    rev = "${pname}-${version}";
-    sha256 = "sha256-7syRpCNL7TRZoJaNrmAdahNy7IyovyniYyOwD/lzhuw=";
+    repo = "cbmc";
+    rev = "cbmc-${finalAttrs.version}";
```

Comment:
-> `rev = "refs/tags/cbmc-${finalAttrs.version}";` (to prevent branch/tag mismatch).